atlp-rwanda/git-practices-falcons
# #0006. Add An md file of a movie summary #6

### What does this PR do?

1.   Add a md file describing a movie
2.   Add a line in Readme file descibing the md file added

#### Description of Task to be completed?

  Summary description of one of the popular telenovella movie which is among my favourites

#### How should this be manually tested?
         
          git clone https://github.com/atlp-rwanda/git-practices-destructors.git

 and check out to my branch  as below 

           
             git checkout ch-update-readme-0006
## Authors

👤 **Divine Charlotte**

- GitHub: [@Divine](https://github.com/divinecharlotte)
- LinkedIn: [@Divine Charlotte](https://www.linkedin.com/in/charlotte-divine-dusenge/)
